dora/tests/integration/src/test/java/alluxio/client/fs/io/FileOutStreamIntegrationTest.java

```diff
@@ -227,31 +240,4 @@ public void cancelWrite() throws Exception {
       Assert.assertEquals(0, worker.getUsedBytes());
     }
   }
-
-  @Test
-  public void getStatusBeforeClose() throws Exception {
```

Comment:
Can you explain the reason of this deletion?

Reply:
have checked with mai, this test is no longer needed